internal/worker/client/command/start.go

```diff
@@ -0,0 +1,41 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/renatoaguimaraes/job-scheduler/internal/worker/proto"
+	"google.golang.org/grpc"
+)
+
+type StartCommand struct {
+	client proto.WorkerServiceClient
+	args   []string
+}
+
+func NewStartCommand(client proto.WorkerServiceClient) Runner {
+	return &StartCommand{
+		client: client,
+	}
+}
+
+func (c *StartCommand) Init(args []string) {
+	c.args = args
+}
+
+func (c *StartCommand) Run() {
```

Comment:
same arguments as above - increase timeout, add length check for args, indent error flow and exit with non-0 status on error